### PR TITLE
explain default value of output.chunkFormat and output.chunkLoading is inferred from target

### PR DESCRIPTION
**Pull request by tanhauhau**
related #5917

### CODE REVIEW

File: src/content/configuration/output.mdx [hunk 1]
```diff
@@ -172,6 +172,8 @@ module.exports = {
 
 The format of chunks (formats included by default are `'array-push'` (web/WebWorker), `'commonjs'` (node.js), `'module'` (ESM), but others might be added by plugins).
 
+T> The default value of this option is depending on the [`target`](/configuration/target/) and [`output.module`](#outputmodule) setting. For more details, search for `"workerChunkLoading"`: [in the webpack defaults](https://github.com/webpack/webpack/blob/master/lib/config/defaults.js).
```
Comment (chenxsan) [inline src/content/configuration/output.mdx:175]: You meant searching `chunkFormat` here?

File: src/content/configuration/output.mdx [hunk 2]
```diff
@@ -226,6 +228,8 @@ module.exports = {
 
 The method to load chunks (methods included by default are `'jsonp'` (web), `'import'` (ESM), `'importScripts'` (WebWorker), `'require'` (sync node.js), `'async-node'` (async node.js), but others might be added by plugins).
 
+T> The default value of this option is depending on the [`target`](/configuration/target/) setting. For more details, search for `"workerChunkLoading"`: [in the webpack defaults](https://github.com/webpack/webpack/blob/master/lib/config/defaults.js).
```
Comment (chenxsan) [inline src/content/configuration/output.mdx:231]: I'm confused by this one as well, maybe you meant searching `chunkLoading`?

File: src/content/configuration/output.mdx (hunk 1 above)
Comment (tanhauhau) [reply src/content/configuration/output.mdx]: oops. i copied over the tip from `workerChunkLoading`

File: src/content/configuration/output.mdx [hunk 3]
```diff
@@ -2289,7 +2293,7 @@ module.exports = {
 
 The new option `workerChunkLoading` controls the chunk loading of workers.
 
-T> The default value of this option is depending on the `target` setting. For more details, search for `"workerChunkLoading"`: [in the webpack defaults](https://github.com/webpack/webpack/blob/master/lib/config/defaults.js).
+T> The default value of this option is depending on the `target` setting. For more details, search for `"workerChunkLoading"`: [in the webpack defaults](https://github.com/webpack/webpack/blob/main/lib/config/defaults.js).
```
Comment (chenxsan) [inline src/content/configuration/output.mdx:2296]:
```suggestion
T> The default value of this option depends on the [`target`](/configuration/target/) setting. For more details, search for `"workerChunkLoading"` [in the webpack defaults](https://github.com/webpack/webpack/blob/main/lib/config/defaults.js).
```

**Review by chenxsan — APPROVED** [review]
Thanks.

**Pull request merged**